src/category_theory/equivalence.lean

```diff
@@ -61,9 +61,12 @@ universes v₁ v₂ v₃ u₁ u₂ u₃
   a unit and counit which are natural isomorphisms and the triangle law `Fη ≫ εF = 1`, or in other
   words the composite `F ⟶ FGF ⟶ F` is the identity.
 
-  In `unit_inverse_comp`, we show that this is actually an adjoint equivalence, i.e., that the
+  In `functor_unit_iso_comp'`, we show that this is actually an adjoint equivalence, i.e., that the
```

Comment:
I don't think this correction is accurate. `functor_unit_iso_comp'` is the triangle law mentioned on line 61, which is part of the constructor, and `unit_inverse_comp` is the proof of the other triangle law `G ⟶ GFG ⟶ G`.

Reply:
Argh, you're right, I didn't read this very well. Sorry.